boot/bootdata/hivesys.inf

```diff
@@ -1576,18 +1576,18 @@ HKLM,"SYSTEM\CurrentControlSet\Services\EventLog","Type",0x00010001,0x00000010
 
 HKLM,"SYSTEM\CurrentControlSet\Services\EventLog\Application",,0x00000010
 HKLM,"SYSTEM\CurrentControlSet\Services\EventLog\Application","File",0x00020000,"%SystemRoot%\system32\config\AppEvent.Evt"
-HKLM,"SYSTEM\CurrentControlSet\Services\EventLog\Application","MaxSize",0x00010003,5242880
-HKLM,"SYSTEM\CurrentControlSet\Services\EventLog\Application","Retention",0x00010003,0
```

Comment:
Please don't change the default retention value yet:
https://git.reactos.org/?p=reactos.git;a=commit;h=37afd9c3718551fa004633f883dc195710ae4ba2
This is because, as ppl currently extensively run/restart ReactOS, the logs would quickly fill up and testers wouldn't be able to see newer entries (until the timeout set by the retention elapses).
However when ReactOS will be much more stable etc..., some reasonable retention values could be set back again.

Reply:
Is your comment on the retention value (7 days) or on the max size (512 Kb) ?

Or more globally is it to have 5 Mb max size and no purge (eg : "replace when necessary") ?

FYI : logs are currently sized to max size, even if empty. I'll look why ater.

Reply:
OK, implemented